src/test/java/com/fitwsarah/fitwsarah/accountsubdomain/presentationlayer/AccountControllerUnitTest.java

```diff
@@ -1,16 +1,59 @@
 package com.fitwsarah.fitwsarah.accountsubdomain.presentationlayer;
 
 import com.fitwsarah.fitwsarah.accountsubdomain.businesslayer.AccountService;
+import com.fitwsarah.fitwsarah.fitnesspackagesubdomain.presentationlayer.FitnessPackageController;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.autoconfigure.web.reactive.WebFluxTest;
 import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
 import org.springframework.test.web.reactive.server.WebTestClient;
 
 import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.when;
+
 @WebFluxTest(controllers = AccountController.class)
 class AccountControllerUnitTest {
     @Autowired
     WebTestClient webTestClient;
     @MockBean
     AccountService accountService;
-}
\ No newline at end of file
+    @InjectMocks
+    private AccountController accountController;
+    //THE ID BELOW WORKS ON EMILE'S COMPUTER WILL HAVE TO FIX W/E IS WRONG WITH MINE
+    private String testToken = "Bearer eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6IjlBNEZQeXE4LTh1OVZqR1IycmJQTiJ9..h---_3EE-0gQ6oKpSQm1-85_icKtLH40u8S6GR9stSCBoBwI7tuaHcX0TJ47QKvRGl4zqY9LxvCrEJ392MFAVn0te-BOmUQgEqzG9WiYz-87psSx8Q1GDPR8kDnmBx6fPTwexxTWPUPgc3dyeR2j5BGfjl4qJp0hk0ZH1h61FLk75Cx-ojFis2UVWbswmPqIj_o0bstxCwle4UDoyr5PIb88vE0_BTmx1WtW-kjZguzcWhXX7U1aB9fD18Qw7g";
```

Comment:
WHy did implement the test token into your test it looks like a smart thing to do?

Reply:
Good question! We need this test token in the header to test the API calls while having the right authorization to do so.